llm-server/src/model/remote-model-factory.ts

```
@@ -0,0 +1,34 @@
+    }
+    return this.instances.get(key)!;
+  }
+
+  static isRemoteModel(config: ModelConfig): boolean {
+    return Boolean(config.endpoint && config.token);
+  }
+
+  static reset(): void {
+    this.instances.clear();
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)